src/io/pti.jl

```diff
@@ -1135,10 +1149,10 @@ function export_pti(io::IO, data::Dict{String,Any})
         # Skip transformers and put it in transformers Array
         if branch["transformer"]
 
-            if data["source_type"] == "matpower"
-                k = 0
-            else 
+            if get(data, "source_type", "matpower") == "pti"
```

Comment:
Nit, A default value of `""` or `"undefined"` would be preferable to `"matpower"`.  If no `"source_type"` is given then it is not clear where the data originally